### ISSUE TITLE
[Monitor] Show error when GPX file is too large

### ISSUE THREAD

**Issue by hgcvm** [vmarc/knooppuntnet#303]
I received a GXP file of 17 MB. (strangely, the file had tenthousands of useless (broken) links to geotagged images).
The upload failed. That's OK, 17 MB is really too big. But I didn't see any error message.

**Comment by vmarc** [comment]
@hgcvm can you try to send the gpx file to me by email?

**Comment by vmarc** [comment]
@hgcvm thanks for providing the gpx file for testing

When trying to upload this file, the response in the browser is:
```
413 Request Entity Too Large
```

The nginx logfile has an error message that contains:
```
client intended to send too large body: 17739584 bytes
```

I have changed the application configuration and the nginx configuration to allow for messages with size up to 20MB.  After this change the file can be uploaded.

Not closing this issue yet: the error handling in the browser application has to be improved so that a proper error message is displayed when the new limit of 20MB is exceeded, or some other problem occurs during file upload.

**Comment by vmarc** [comment]
Fixed in v4.2.2.
